### ISSUE TITLE
serverless config credentials fails to create .aws directory in windows 10

### ISSUE THREAD

**Issue by paradigmist** [serverless/serverless#2857]
# This is a Bug Report

## Description

For bug reports:
* What went wrong?
`serverless config credentials` failed to create `.aws/` directory to save `credentials` file. The issue solved after manually create `.aws` folder.

* What did you expect should have happened?
`serverless config credentials` should be able to create `.aws/` at home directory.

* What was the config you used?
The bug comes before project creation.

* What stacktrace or error message from your provider did you see?
```
Serverless: Setting up AWS...
Serverless: Saving your AWS profile in "~/.aws/credentials"...
Serverless: Success! Your AWS access keys were stored under the "default" profile.

  Error --------------------------------------------------

     ENOENT: no such file or directory, open 'C:\Users\FONG\.aws\credentials'

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues

     Please report this error. We think it might be a bug.

  Your Environment Information -----------------------------
     OS:                 win32
     Node Version:       6.9.1
     Serverless Version: 1.3.0
```

## Additional Data

* ***Serverless Framework Version you're using***: 1.3.0
* ***Operating System***: windows 10
* ***Stack Trace***:
* ***Provider Error messages***:

**Comment by pmuens** [comment]
@paradigmist thanks for reporting. Do you know where your AWS credentials are stored usually (if you've already used e.g. the AWS CLI).

Where is your home directory located?

**Comment by paradigmist** [comment]
@pmuens If I used AWS CLI, the credentials are usually stored at C:\Users\NAME.

In windows, my home directory is C:\Users\NAME

**Comment by pmuens** [comment]
@paradigmist thanks for getting back!

I'll close this one as we're continuing the discussion in #2878 
We'll work on a fix which should be available in 1.4.

Thanks again for bringing this up!
